tests/test_sqldiff.py

```diff
@@ -6,23 +6,27 @@
 from django.test import TestCase
 
 # from django.core.management import call_command
-from django_extensions.management.commands.sqldiff import SqliteSQLDiff, Command
+from django_extensions.management.commands.sqldiff import SqliteSQLDiff, Command, MySQLDiff, PostgresqlSQLDiff
 
 
 class SqlDiffTests(TestCase):
+
+    def setUp(self):
+        self.parser = Command().create_parser("test", "sqldiff")
+        self.args = ["-a"]
+        self.options = self.parser.parse_args(args=self.args)
+        self.tmp_out = six.StringIO()
+        self.tmp_err = six.StringIO()
+
     def _include_proxy_models_testing(self, should_include_proxy_models):  # type: (bool) -> ()
-        parser = Command().create_parser("test", "sqldiff")
-        args = ["-a"]
         if should_include_proxy_models:
-            args.append("--include-proxy-models")
-        options = parser.parse_args(args=args)
-        tmp_out = six.StringIO()
-        tmp_err = six.StringIO()
+            self.args.append("--include-proxy-models")
```

Comment:
This should be `args` not `self.args` ?
Now we keep appending to `self.args` which each test executed in the class.

Reply:
@trbs I assumed that SetUp is executed for each tests method case separately.

Reply:
I only need that in this case only therefore only appended here.

Reply:
Your absolutely right. It's called before every test... makes sense to otherwise `tmp_out` and `tmp_err` would have been problematic as well.

(Guess in the end I would probably prefer to rewrite these into `pytest` style tests but that's for another day :-) )